packages/worker/src/worker.ts

```diff
@@ -176,10 +176,10 @@ export interface WorkerOptions {
   maxConcurrentWorkflowTaskExecutions?: number;
```

Comment:
Should this still be so high? Might make sense to crank the default down here a bit too.

Reply:
See comment above, thanks for pointing this out.